webrtc.html

```diff
@@ -1586,22 +1586,20 @@
                         "applying-a-remote-desc">[[!JSEP]]</span>.</p>
                       </li>
                       <li>
-                        <p>If the direction of the media description as defined
-                        in
-                        <span data-jsep="applying-a-remote-desc">[[!JSEP]]</span>
-                        is <code>sendrecv</code> or <code>sendonly</code>, and
+                        <p>If <var>direction</var> is <code>sendrecv</code> or
+                        <code>recvonly</code>, and
                         <var>transceiver</var>'s
                         [[<a>CurrentDirection</a>]] slot
-                        is neither "sendrecv" nor "sendonly",
+                        is neither "sendrecv" nor "recvonly",
```

Comment:
This is fine though (remember, "direction" above is the reversed direction from a remote description, whereas CurrentDirection isn't... I wonder if we could do something to make this less confusing).